invenio_records_rest/utils.py

```diff
@@ -50,14 +55,20 @@ def obj_or_import_string(value, default=None):
 
 
 def load_or_import_from_config(key, app=None, default=None):
-    """Load or import value from config."""
+    """Load or import value from config.
+
+    :returns: A :class:`invenio_access.permissions.DynamicPermission` object
```

Comment:
The description is not correct. It can return anything stored in config under defined key or if it is a string then it tries to import it.